### DISCUSSION TITLE
Page Footer

### DISCUSSION THREAD

**Discussion by Rozerinay** [nl-design-system/backlog#282] [Component suggestions]
<img width="720" alt="visuele weergave van de page footer component" src="https://github.com/user-attachments/assets/f5e2df05-27c4-41c9-9fa2-51943c37fc18">

We willen graag meer leren over de **Page Footer** component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook, Github of screenshots. Met deze informatie kunnen we bewijzen dat de component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1733486460820259)

---

## Naam
Page Footer

## Doel
Consistente plek onderaan elke pagina, vaak met links, juridische informatie en opties om hulp te krijgen of contact op te nemen.

## Ook bekend als
Footer

## Gerelateerde componenten
[Root](https://github.com/orgs/nl-design-system/discussions/426), [Body](https://github.com/orgs/nl-design-system/discussions/427), [Page Layout](https://github.com/orgs/nl-design-system/discussions/428), [Page Header](https://github.com/orgs/nl-design-system/discussions/283), [Page Body](https://github.com/orgs/nl-design-system/discussions/429)

## Gerelateerde patronen
[Page Footer (Hulp bieden) - Meerstappenformulier](https://github.com/orgs/nl-design-system/discussions/383)

## Onderzoek

[Web Page Footers 101: Design Patterns and When to Use Each - NN/g](https://www.nngroup.com/articles/footers/)
Het artikel bespreekt de evolutie en het belang van Footers. 

[Footers Are Underrated - NN/g](https://www.nngroup.com/videos/footers/)
Uit gebruikerstests blijkt dat gebruikers vaak naar Footers kijken voor belangrijke informatie en taken, waardoor ze een essentieel onderdeel vormen van de algehele ervaring van een site.

[Footer Links Should be Divided into Distinct Semantic Sections - Baymard Institute](https://baymard.com/blog/footer-links-ecommerce)
Het gebruikersonderzoek toont aan dat Footers vaak een laatste redmiddel zijn voor gebruikers om informatie zoals retourbeleid of live chat te vinden, maar dat slecht ontworpen Footers vaak tot frustratie en gemiste kansen leiden. Het benadrukt hoe duidelijke, semantische secties en aangepaste ontwerpen voor mobiel en desktop essentieel zijn om gebruikers effectief te ondersteunen.

[GOV.UK footer - GOV.UK](https://design-system.service.gov.uk/components/footer/)
Footer implementatie bij GOV.UK. Bij de [Footer Discussion](https://github.com/alphagov/govuk-design-system-backlog/issues/96) wordt meer informatie, onderzoek en aanpassingen gedeeld. Enkele interessante comments:

- Er is vaak een inconsistente volgorde en benaming van Links in Footers (zoals Contact vs. Contact us), wat verwarrend kan zijn voor gebruikers. Volgens [WCAG 2.2’s succescriterium 3.2.6](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html), dat consistentie in het presenteren van informatie over hulp benadrukt, zou het nuttig zijn als richtlijnen voor Footers een standaard volgorde en uniforme naamgeving van Links vastleggen, bijvoorbeeld alfabetisch, om de gebruikservaring te verbeteren.
- Links naar ‘help’-pagina’s in de Footer moeten consistent worden geplaatst en uniform functionerend zijn op alle pagina’s. Dit is nodig om te voldoen aan [WCAG 2.2 succescriterium 3.2.6](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html), dat consistentie in hulpinformatie voorschrijft.
- Bij het toevoegen van een logo moet er rekening gehouden worden met WCAG 1.4.11: Non-text Contrast met een contrast van 4.08:1 (hoger dan de vereiste 3:1).

[ARIA: contentinfo role - MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/contentinfo_role)
De contentinfo role verbetert de navigatie voor gebruikers van ondersteunende technologieën en maakt het eenvoudiger om snel naar de footer-informatie te springen, wat de algehele gebruikerservaring verbetert.

[Page Regions - W3C](https://www.w3.org/WAI/tutorials/page-structure/regions/)
De region rol kan worden gebruikt als een bredere aanduiding van een belangrijk gebied op de pagina, maar is minder specifiek dan contentinfo. Dit helpt bij het markeren van de Footer als een belangrijk, afzonderlijk gebied van de pagina, zodat gebruikers het makkelijk kunnen vinden.

**Comment by Rozerinay** [comment]
## Amsterdam

- [Storybook](https://designsystem.amsterdam/?path=/docs/components-containers-footer--docs)
- [Figma](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?node-id=2927-29216&node-type=canvas&t=RYNmSkQ5HUM3QD0u-11)
- [GitHub](https://github.com/Amsterdam/design-system/tree/develop/packages/css/src/components/footer)

---

### Screenshots / Varianten

**Storybook**

Default
<img width="800" alt="image" src="https://github.com/user-attachments/assets/8b812287-4a6c-4f45-afe2-36be586c1c3d">

Onderzoek en Statistiek
The footer for a specific website can be a bit different. Here’s an example for the ‘Onderzoek en Statistiek’ department.

<img width="800" alt="image" src="https://github.com/user-attachments/assets/7928e236-492d-4d77-9cf2-95810a4f6fb3">

Responsive gedrag
<img width="360" alt="image" src="https://github.com/user-attachments/assets/044161e0-abc5-4f2c-9f25-8c5696567d27">

---

### Notities

- Onder de naam: Footer.
- Bestaat uit een Spotlight en Page Menu (waar privacy en cookies te vinden zijn).
- Spotlight bevat weer een Link List, Paragraph en Headings.
- Is verdeeld over 3 kolommen.
- Heeft een gekleurde (donkere) achtergrond met witte content.
- Responsive gedrag: aantal kolommen vermindert (3, 2, 1) en inhoud komt onder elkaar te staan. 
- Richtlijnen beschikbaar.

**Comment by Rozerinay** [comment]
## Den Haag

- [Storybook](https://nl-design-system.github.io/denhaag/?path=/story/css-page-elements-footer--default)
- [Figma](https://www.figma.com/design/JpoY3waVoQGlLQzQXTL9nn/%F0%9F%8C%BB-Den-Haag-Design-System?node-id=9024-50096&node-type=frame&t=mKin8dFYtdOt45TX-11)
- [GitHub](https://github.com/nl-design-system/denhaag/tree/main/components/Footer)

---

### Screenshots / Varianten

**Storybook**

<img width="800" alt="image" src="https://github.com/user-attachments/assets/f11738a5-f055-4f4b-980f-d8a81f83aac9">

Responsive gedrag
<img width="320" alt="image" src="https://github.com/user-attachments/assets/c3b844ad-53cc-455d-98cb-fd2df014754b">
<img width="320" alt="image" src="https://github.com/user-attachments/assets/462d5c50-5bfc-492d-bfda-b72ae6e820bd">

**Figma**

Megafooter

<img width="800" alt="image" src="https://github.com/user-attachments/assets/f7e6c0a5-dd44-4c03-ab09-a23b9a914d7a">

Responsive gedrag
<img width="300" alt="image" src="https://github.com/user-attachments/assets/1d0295c4-e393-4981-af68-1effca4ebcb4">

Contact footer

<img width="800" alt="image" src="https://github.com/user-attachments/assets/d131acb5-0b76-4787-b772-f77777aa27da">

Responsive gedrag

<img width="300" alt="image" src="https://github.com/user-attachments/assets/f6f91d36-df33-4616-9415-032d24133a29">

Legal footer

<img width="765" alt="image" src="https://github.com/user-attachments/assets/f9fa9462-956f-4818-9071-87557f91aedb">

Responsive gedrag

<img width="300" alt="image" src="https://github.com/user-attachments/assets/bd920812-b5ce-4699-b62f-649055210af1">


---

### Notities

- Onder de naam: Footer.
- Footer group is ingedeeld in 3 soorten footers, namelijk: mega, contact, legal.
- Bestaat uit Headings, Link List, Button, List Items en een Paragraph.
- Is verdeeld over 3 kolommen.
- Heeft een gekleurde (donkere) achtergrond met witte content.
- Responsive gedrag: aantal kolommen vermindert (3, 2, 1) en inhoud komt onder elkaar te staan.
- Figma: op small container zijn de list items van de mega footer onderverdeeld in een uitklapper (alleen in design).
- Richtlijnen beschikbaar (Figma).

**Reply by eslook** [reply]
Update van @thijsren: https://github.com/orgs/nl-design-system/discussions/282#discussioncomment-16209381

**Comment by Rozerinay** [comment]
## DSO

- [Storybook](https://storybook.dso-toolkit.nl/67.0.0/?path=/story/voorbeeldpagina-s-patronen-footer--footer)
- [GitHub](https://github.com/dso-toolkit/dso-toolkit/tree/master/packages/dso-toolkit/src/components/footer)

---

### Screenshots / Varianten

**Storybook**
<img width="800" alt="image" src="https://github.com/user-attachments/assets/4e206683-c257-44de-9962-72eed789c609">

Responsive gedrag
<img width="320" alt="image" src="https://github.com/user-attachments/assets/7c8c147d-a66c-46f6-9557-40ea2413e595">

---

### Notities

- Onder de naam: Footer.
- Bestaat uit Headings, Link List, Link en een Paragraph.
- Is verdeeld over 4 kolommen.
- Heeft een gekleurde (lichte) achtergrond met donkere (zwarte) content.
- Responsive gedrag: aantal kolommen vermindert (4, 2, 1) en inhoud komt onder elkaar te staan.

**Comment by Rozerinay** [comment]
## DUO

[Documentatie website](https://uno.dfront.rijkscloud.nl/#/components/footer)

---

### Screenshots / Varianten

**Documentatie website**
<img width="736" alt="image" src="https://github.com/user-attachments/assets/4b41e20e-6a28-4f29-a845-0ce0c66f3d39">

Responsive gedrag
<img width="320" alt="image" src="https://github.com/user-attachments/assets/65d20528-e9f6-4191-9552-ab79f04bdd19">

---

### Notities

- Onder de naam: Footer.
- Bestaat uit een pay-off, Headings en Links. 
- Is verdeeld over 1 rij die weer bestaat uit 2 kolommen.
- Heeft een gekleurde (donkere) achtergrond met witte content.
- Responsive gedrag: aantal kolommen vermindert (2, 1) en inhoud komt onder elkaar te staan.
- Richtlijnen beschikbaar.

**Comment by Rozerinay** [comment]
## OpenGemeenten

- [Documentatie website](https://arc.net/l/quote/pwsrnzvw)
- [Patternlab](https://patternlab.lenteveld.nl/patterns/02-organisms-999-Page-Footer-OpenGemeentenFooter-PageFooter/02-organisms-999-Page-Footer-OpenGemeentenFooter-PageFooter.html)

---

### Screenshots / Varianten

**Patternlab**

<img width="800" alt="image" src="https://github.com/user-attachments/assets/2442a2fe-e459-4fa4-9adb-ad08614d747b">

Page Footer Link
<img width="800" alt="image" src="https://github.com/user-attachments/assets/1334a1b7-62e5-45d6-9b94-0f6e9e908116">

Page Footer Orientation Rows
<img width="800" alt="image" src="https://github.com/user-attachments/assets/57964cd3-b3e4-4ccd-a1aa-6719ed585d99">

Page Footer Orientation Rows Long
<img width="800" alt="image" src="https://github.com/user-attachments/assets/f87fd77d-c3dc-41ac-acb5-0b837a5cccdd">

Responsive gedrag

<img width="320" alt="image" src="https://github.com/user-attachments/assets/1500e69c-6634-4258-b8ed-ff60a6a10ad9">


Voorbeelden verschillende gemeentes

Gemeente Schouwen-Duiveland
<img width="800" alt="image" src="https://github.com/user-attachments/assets/340e03c2-7628-46d9-b06a-83090cf49785">

Responsive gedrag
<img width="320" alt="image" src="https://github.com/user-attachments/assets/af9bcaa7-528b-47b0-9440-9a0bfdd031e7">

Gemeente Westland
<img width="800" alt="image" src="https://github.com/user-attachments/assets/b46a26d7-140a-4d00-adf3-d8f0c95aace9">

Responsive gedrag
<img width="320" alt="image" src="https://github.com/user-attachments/assets/922ed64e-2ea8-4a4c-b33b-bfaea92f04c4">

---

### Notities

- Onder de naam: Page Footer.
- Bestaat uit Headings, Button, Address, Link List en Links.
- Kan bestaan uit een logo (rechts gepositioneerd). 
- Is verdeeld over 3 kolommen.  Kan in de variant ‘Orientation Rows’ ook bestaan uit meerdere rijen.
- Heeft een gekleurde (donkere) achtergrond met witte content.
- Responsive gedrag: aantal kolommen vermindert (3, 1) en inhoud komt onder elkaar te staan.

**Comment by Rozerinay** [comment]
## Rotterdam

- [Figma](https://www.figma.com/design/iWuQsxelJzXVfCge87Og81/RODS-component?node-id=2-1478&node-type=frame&t=a2u80f8m17PWqVgu-11)
- [GitHub](https://github.com/nl-design-system/rotterdam/tree/main/rotterdam-nlds-parent-wicket/docs-wicket/src/main/java/nl/rotterdam/nl_design_system/docs/wicket/page_footer)

---

### Screenshots / Varianten

**Figma**

White
<img width="1274" alt="image" src="https://github.com/user-attachments/assets/a29a58f6-f6df-47ae-93aa-471b4685b539">

Green
<img width="1273" alt="image" src="https://github.com/user-attachments/assets/d6800d56-a00c-4f06-a0c8-4910ebfefd97">

**Publieke website**
<img width="1415" alt="image" src="https://github.com/user-attachments/assets/cdafb445-f6b5-43a6-b048-6110a7d602c0">

Responsive gedrag
<img width="366" alt="image" src="https://github.com/user-attachments/assets/cb876f5d-8a02-47b3-9322-c47e9c7dedb5">

---

### Notities

- Onder de naam: Footer.
- Bestaat uit Links, Separator en een Paragraph.
- Kan bestaan uit een logo (rechts gepositioneerd).
- Is verdeeld over meerdere rijen.
- Heeft een gekleurde (donkere) achtergrond met witte content.
- Responsive gedrag: Links hebben overflow wrap en komen dus op de volgende regel te staan.

**Comment by Rozerinay** [comment]
## RVO

- [Documentatie website](https://nl-design-system.github.io/rvo/docs/componenten/footer/)
- [Storybook](https://nl-design-system.github.io/rvo/?path=/story/componenten-footer--default)
- [Figma](https://www.figma.com/design/Sj6myBL1Fvot5M1qGxzvEo/ROOS-(RVO-Design-System)?node-id=287-6733&node-type=canvas&t=nSK654olX49sPmgT-11)
- [GitHub](https://github.com/nl-design-system/rvo/tree/master/components/footer)

---

### Screenshots / Varianten

**Storybook**
<img width="800" alt="image" src="https://github.com/user-attachments/assets/b9b740b4-556a-47ab-8931-018f2c1c32d7">

Responsive gedrag
<img width="500" alt="image" src="https://github.com/user-attachments/assets/83e71152-f81f-45cd-b287-c18542cd575e">

<img width="300" alt="image" src="https://github.com/user-attachments/assets/bf65d49e-d621-4a4c-9ffc-4bfe281a3f62">

---

### Notities

- Onder de naam: Footer.
- Bestaat uit Headings en Links.
- Kan bestaan uit een optionele pay-off.
- Is verdeeld over 4 kolommen.
- Heeft een gekleurde (donkere) achtergrond met witte content.
- Responsive gedrag: aantal kolommen vermindert (4, 3, 2, 1) en inhoud komt onder elkaar te staan.
- Richtlijnen beschikbaar (documentatie website).

**Comment by Rozerinay** [comment]
## Utrecht

- [Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-page-footer--docs)
- [GitHub](https://github.com/nl-design-system/utrecht/tree/main/components/page-footer)

---

### Screenshots / Varianten

**Storybook**
<img width="800" alt="image" src="https://github.com/user-attachments/assets/fe4bf8c7-4385-4e4a-8248-f926dfc7cf76">

Responsive gedrag
<img width="300" alt="image" src="https://github.com/user-attachments/assets/26850490-c05f-4f7e-96db-1992f491be96">

---

### Notities

- Onder de naam: Page Footer.
- Bestaat uit Headings, Paragraph en Links.
- Is verdeeld over meerdere rijen.
- Heeft een gekleurde (donkere) achtergrond met witte content.
- Responsive gedrag: inhoud blijft onder elkaar staan. Geen verandering. 
- Richtlijnen beschikbaar (documentatie website).

**Comment by Rozerinay** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Page Footer op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=page+footer)

**Comment by TessaViergever** [comment]
Zie mijn comment bij de [Link List](https://github.com/orgs/nl-design-system/discussions/266#discussioncomment-14163942) discussion :)

**Comment by eslook** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Page Footer op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=page+footer)

**Comment by thijsren** [comment]
Ik zie hierboven in de thread een oudere versie van de footer uit Den Haag. De laatste versie staat [hier](https://www.figma.com/design/eSl0Sclc8t03X76qM1cWyz/%F0%9F%93%95-HDS-Library?node-id=2913-26825) 

<img width="1960" height="2816" alt="denhaag-footer" src="https://github.com/user-attachments/assets/10505fcd-ac46-4f68-8044-4042560e4ec5" />

**Reply by eslook** [reply]
Dankjewel voor de update @thijsren !
